woocommerce/woocommerce-ios
# [Analytics Hub] Connect data and UI for Sessions analytics card

Closes: #8365

## Description

This PR connects the data used for the sessions card (current period order stats and site summary stats) with the card UI.

For now, this will display placeholders and a sync error because we aren't fetching real data yet. The data will be fetched in another PR.

## Changes

* Adds a `siteStats` property to store the fetched `SiteSummaryStats` (currently always `nil`).
* Adds a helper that returns a `AnalyticsReportCardCurrentPeriodViewModel` for the sessions card.
* Localizes the card strings.

## Testing

1. Build and run the app.
2. Tap "See more" to open the Analytics Hub.
3. Confirm the Sessions card displays real data (placeholders and sync error for now).

## Screenshots

<img src="https://user-images.githubusercontent.com/8658164/207573185-66d68a58-5235-484a-bf8d-983eb0ca3253.png" width="300px">


## Submitter Checklist

Update release notes:
